comps/llms/text-generation/vllm/docker_compose_llm.yaml

```diff
@@ -7,30 +7,30 @@ services:
   vllm_service:
     image: vllm:cpu
     container_name: vllm-service
+    network_mode: host
     ports:
       - "8008:80"
     volumes:
       - "./data:/data"
     environment:
       http_proxy: ${http_proxy}
       https_proxy: ${https_proxy}
+      no_proxy: ${no_proxy}
       LLM_MODEL_ID: ${LLM_MODEL_ID}
       HF_TOKEN: ${HUGGINGFACEHUB_API_TOKEN}
-    command: cd / && export VLLM_CPU_KVCACHE_SPACE=40 && python3 -m vllm.entrypoints.openai.api_server --model $LLM_MODEL_ID --port 80
+    command: /bin/sh -c "cd / && export VLLM_CPU_KVCACHE_SPACE=40 && python3 -m vllm.entrypoints.openai.api_server --host 0.0.0.0 --model $LLM_MODEL_ID --port 80"
```

Comment:
if removing '-host' option here, any functionity impact?

Reply:
@ftian1 If I removed this option, the container occurred the "Internal Server Error" as shown in the below:
![image](https://github.com/opea-project/GenAIComps/assets/87695601/0f4957fd-4c60-484f-8bfb-4ddf0123e5e6)

Reply:
@ftian1 I have removed the "-host", then we can test by using the actual IP instead of "127.0.0.1".
![image](https://github.com/opea-project/GenAIComps/assets/87695601/6bc68b8c-feb6-4f27-9bc6-94f632dc7a67)